devu-shared/src/types/nonContainerAutoGrader.types.ts

```diff
@@ -0,0 +1,9 @@
+export type NonContainerAutoGrader = {
+    id: number
```

Comment:
id should be optional (with a ?) since when creating an entity on the front end, there will not yet be an id